docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+services:
+
+  picard_website:
+    build:
+      context: ..
+      dockerfile: ./Dockerfile
+    ports:
```

Comment:
I don't think this should be a public port on the host, using `expose` instead of `ports` would be enough to make the port available to the nginx container.